### PR TITLE
Improve justified container for older browsers including safari

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
Fixes the justified container component for older browsers including safari - https://github.com/holidayextras/ui-toolkit/issues/55

#### What tests does this PR have?
None - i've actually removed one of the tests which checks for inline styles

#### How can this be tested?
run `grunt docs` and check justified container in safari.
compare www.ui-toolkit.com with your local version on safari. you will see that the justified conainter now works fully.

#### Screenshots / Screencast
screen shot 2015-08-14 at 12 57 01
screen shot 2015-08-14 at 12 56 01

#### What gif best describes how you feel about this work?
I'm too flexy

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

### CODE REVIEW

**Comment by HeinBehrenshx** [comment]
taking se/dev

**Comment by HeinBehrenshx** [comment]
+1

**Comment by hxjamesmcneil** [comment]
SE/Guru

**Comment by hxjamesmcneil** [comment]
Looks good :shipit:

File: src/less/toolkit.less [hunk 1]
```diff
@@ -26,11 +26,6 @@
 @fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts"; // for referencing Bootstrap CDN font files directly
 
 //
-// == Flex-Less ==
-//
-@import "../../node_modules/flex-less/flex";
```
Comment (Joezo) [inline src/less/toolkit.less]: can we remove flex-less from package json now?

File: docs/examples/JustifiedContainer.jsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 var example = (
   <UIToolkit.JustifiedContainer>
-    <UIToolkit.Button size="block">foo</UIToolkit.Button>
-    <UIToolkit.Button size="block">bar</UIToolkit.Button>
-    <UIToolkit.Button size="block">baz</UIToolkit.Button>
+    <div><UIToolkit.Button size="block">foo</UIToolkit.Button></div>
```
Comment (Joezo) [inline docs/examples/JustifiedContainer.jsx]: Why are these in divs now?

**Pull request merged**